### DISCUSSION TITLE
galaxy brain- I will help

### DISCUSSION THREAD

**Discussion by pranjal779** [Schweinepriester/github-profile-achievements#588] [Q&A] [answered]
comment to get marked answer
help me if possible

**Comment by SusheelThapa** [comment, marked as answer]
When our two answer is marked as valid answer then we get Galaxy brain badge

**Reply by pranjal779** [reply]
sure why not

**Reply by pranjal779** [reply]
@MasterSusheel ping me if you plan to help me 🙏 :wink:

**Reply by SusheelThapa** [reply]
@pranjal779  sure i will help me just ping me too

**Reply by pranjal779** [reply]
did you get the badge?

**Reply by SusheelThapa** [reply]
@pranjal779  Yup I got it.. Thanks buddy .. 

If you need my help just ping me
